R-package/tests/testthat/test_dataset.R

```diff
@@ -547,4 +547,15 @@ test_that("lgb.Dataset$get_feature_num_bin() works", {
   )
   actual_num_bins <- sapply(1L:5L, ds$get_feature_num_bin)
   expect_identical(actual_num_bins, expected_num_bins)
+  # test using defined feature names
+  bins_by_name <- sapply(colnames(raw_mat), ds$get_feature_num_bin)
+  expect_identical(unname(bins_by_name), expected_num_bins)
+  # test using default feature names
+  no_names_mat <- raw_mat
+  colnames(no_names_mat) <- NULL
+  ds_no_names <- lgb.Dataset(no_names_mat, params = list(min_data_in_bin = min_data_in_bin))
+  ds_no_names$construct()
+  default_names <- lapply(seq(1L, ncol(raw_mat)), function(i) sprintf("Column_%d", i - 1L))
```

Comment:
```suggestion
  default_names <- lapply(
    X = seq(1L, ncol(raw_mat))
    , FUN = function(i){
      sprintf("Column_%d", i - 1L)
    }
  )
```

Could you please reformat this like this?

It's a personal style preference, but I've found that `lapply()`, `sapply()`, etc. calls in R are easier to read if the function definition uses `{}` and if the body of the function isn't on the same line as the function signature.

Reply:
added in fe6c9b4b6cf0f60af366dc402acaad9c182ceb31